api/files/serializers.py

```diff
@@ -88,6 +111,7 @@ class FileSerializer(JSONAPISerializer):
         'size',
         'provider',
         'last_touched',
+        'tags',
```

Comment:
Could you verify that tags are indexed in ModularODM? I mean, it would be crazy if they weren't, because why tag if you aren't going to be able to search, but just in case.